RealmTasks Apple/RealmTasks iOS/CellPresenter.swift

```diff
@@ -31,36 +31,38 @@ class CellPresenter<Item: Object where Item: CellPresentable> {
         self.items = items
     }
 
-    func deleteItem(item: Item) {
+    func delete(item: Item) {
         viewController.uiWrite {
-            guard !(item as Object).invalidated, let index = items.indexOf(item) else {
+            guard !(item as Object).isInvalidated, let index = items.index(of: item) else {
                 return
             }
 
             items.realm?.delete(item)
-            viewController.tableView.deleteRowsAtIndexPaths([NSIndexPath(forRow: index, inSection: 0)], withRowAnimation: .Left)
+            viewController.tableView.deleteRows(at: [IndexPath(row: index, section: 0)], with: .left)
+            viewController.didUpdateList(reload: true)
```

Comment:
why was this call to `didUpdateList` added? It wasn't there before. Was that a previous omission, or was this added accidentally?